macos/Sources/Features/Terminal/TerminalView.swift

```diff
@@ -31,7 +31,10 @@ protocol TerminalViewModel: ObservableObject {
 
     /// The command palette state.
     var commandPaletteIsShowing: Bool { get set }
-    
+    /// The command palette is reused for the focus palette, this indicates what
+    /// mode it should be in, true for the focus palette.
+    var commandPaletteIsJumpPalette: Bool { get set }
```

Comment:
Replace `commandPaletteIsShowing` with `commandPaletteState` and `CommandPaletteState` is an enum with 3 values: `hidden, command, jump`.